qa/rpc-tests/fundrawtransaction.py

```diff
@@ -316,7 +316,7 @@ def run_test(self):
         rawtx   = self.nodes[2].createrawtransaction(inputs, outputs)
         dec_tx  = self.nodes[2].decoderawtransaction(rawtx)
 
-        assert_raises_jsonrpc(-4, "Insufficient funds", self.nodes[2].fundrawtransaction, rawtx)
+        assert_raises_jsonrpc(-4, "Coin Selection Failed", self.nodes[2].fundrawtransaction, rawtx)
```

Comment:
I agree it'd be better to have more precise error codes/messages returned. However, I think this is an improvement since we're returning "Insufficient funds" when the problem may be something different.